public/js/app.js

```diff
@@ -7,29 +7,32 @@ $(document).ready(() => {
       username: $("input#user-input").val().trim(),
       password: $("input#password-input").val()
     };
-    if ($('#isNewUser').prop('checked')){
+    if ($('#isNewUser').prop('checked')) {
       // user wants to create a new account
-      $.post("/api/signUp",userCredentials).then((res)=>{
+      $.post("/api/signUp", {
```

Comment:
it can be done in one line, its shorter and cleaner
leave as $.post("api/signUp",userCredentials).then(....